Dockerfile

```diff
@@ -162,11 +164,17 @@ COPY --from=locales --chown=olympia:olympia ${HOME}/locale ${HOME}/locale
 # Copy assets from assets
 COPY --from=assets --chown=olympia:olympia ${HOME}/site-static ${HOME}/site-static
 
+# Add build.py build UUID
+RUN ${HOME}/scripts/generate_build.py > build.py
+
 # version.json is overwritten by CircleCI (see circle.yml).
 # The pipeline v2 standard requires the existence of /app/version.json
 # inside the docker image, thus it's copied there.
 COPY version.json /app/version.json
 
+# Set shell back to sh until we can prove we can use bash at runtime
+SHELL ["/bin/sh", "-c"]
```

Comment:
@diox is there a good reason not to keep bash as the default shell for the image?

Reply:
I imagine it's fine to have `bash` as default.

Reply:
Okay. I will open a separate PR for that so we can easier split it if something does go wrong.